couch/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - couch
 
+## 2.7.0 / 2018-10-12
+
+* [Added] Command to validate metadata. See [#2269](https://github.com/DataDog/integrations-core/pull/2269).
```

Comment:
same as above. But since this is the only change, I don't think we should release this. (Nothing thats bundled with the check has changed)